### ISSUE TITLE
Cannot obtain the accuracy stated in the doc for inception_v3 pretrained on Imagenet

### ISSUE THREAD

**Issue by m-parchami** [pytorch/vision#6066]

**Comment by m-parchami** [comment]
Thanks @NicolasHug for moving it to the right place and not just closing it :) Sorry for the incovenience.

Regarding your comment, I assumed that by running `model.eval()` the batch normalization and drop outs can no longer affect the output. Also, I run the code only on a single GPU to avoid any duplication/drops in the samples. Also, because the model is fully pretrained, I assumed that the CUDA determinism can't affect any weights, hence the output.

I appreciate it if you or anyone else can elaborate on how the model can give different results while both the model and the loader are obtained directly from pytorch itself. The preprocess routine was directly copied from the documents.

**Comment by datumbox** [comment]
Actually, I think the documentation on TorchHub needs to change to use the new Multi-weight support API. Instead of manually preprocessing the weights and getting into problems, this should be done using the transforms attached to the weights. Here is an [example](https://pytorch.org/vision/main/models.html#classification) on the docs. 

Probably we want to merge such a PR after the upcoming release though because that's a new API.
